src/Modules/GraphEngine.ServiceFabric/Trinity.ServiceFabric.SampleApplication/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -2,9 +2,9 @@
 <ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="Trinity.ServiceFabric.SampleApplicationType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
   <Parameters>
     <Parameter Name="Trinity.ServiceFabric.GraphEngineRemotingClient_InstanceCount" DefaultValue="-1" />
-    <Parameter Name="Trinity.ServiceFabric.GraphEngineService_MinReplicaSetSize" DefaultValue="3" />
+    <Parameter Name="Trinity.ServiceFabric.GraphEngineService_MinReplicaSetSize" DefaultValue="2" />
```

Comment:
please elaborate why to make this change.

Reply:
@yatli I have done quite a bit of testing trying to understand the policy-behavior of the GE when running in the Service Fabric Cluster Configuration. I'm now running on Azure Stack with Service Fabric configured; I changed these setting because they consistently work when the cluster is in crisis under recovery and or update; with the default configuration the length of time required to restart the services has proven to be problematic. I have not seen any problems with scaling up. I now have 12 GE/SF App Services running serving up some 250,000,000 Hyper-Graph RDF nodes and it is awesome; a 5x(Apps)x5x5x10 configuration works well for my services. Another reason why I changed the cluster configuration was due to the length of time it was taking for services to come ready and if the Sample GE App would ever come back to ready-state.

Here's my CPU and Memory blueprint 

![image](https://user-images.githubusercontent.com/5692812/56984168-4451d680-6b3a-11e9-86bb-63fb8f994466.png)

![image](https://user-images.githubusercontent.com/5692812/56983915-b118a100-6b39-11e9-882c-4cdcdc2b7e0e.png)

Here's the GE/SF apps running very well in the 5x1x2x2x3 configuration. I take cut power or do oderly shutdonw and upgrade (replace) the services and it always comes back in under 2 minutes.

![image](https://user-images.githubusercontent.com/5692812/56984056-02289500-6b3a-11e9-93d4-beb1737ab43b.png)

I am still making a few changes in how the GE App service is written as I am sure that has something to do with it.